### PR TITLE
rules: change rule ids to <HS-LANGUAGE-ID> template

### PR DESCRIPTION
**Pull request by matheusalcantarazup**
Signed-off-by: Matheus Alcantara <matheus.alcantara@zup.com.br>

**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Review by iancardosozup — APPROVED** [review]

**Review by otaviojava — APPROVED** [review]

File: internal/services/engines/csharp/rules.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 func NewCommandInjection() text.TextRule {
 	return text.TextRule{
 		Metadata: engine.Metadata{
-			ID:          "782ad071-1cf3-4230-936f-b7a1e794828d",
+			ID:          "HS-CSHARP-1",
```
Comment (otaviojava) [inline internal/services/engines/csharp/rules.go:31]:
```suggestion
C#
```

Comment (otaviojava) [reply internal/services/engines/csharp/rules.go:31]: @matheusalcantarazup I think this is way is cleaner.
What do you think?

Comment (matheusalcantarazup) [reply internal/services/engines/csharp/rules.go:31]: I personally prefer CSHARP. What do you guys thing? @wiliansilvazup @nathanmartinszup?

Comment (nathanmartinszup) [reply internal/services/engines/csharp/rules.go:31]: Throughout all the Horusec we call it `CSHARP`, maybe it's better keping equal

**Review by nathanmartinszup — APPROVED** [review]

File: internal/services/engines/csharp/rules.go (hunk 1 above)
Comment (wiliansilvazup) [reply internal/services/engines/csharp/rules.go:31]: When horusec show your analysis the language is `Csharp`. So i think it's better to keep `CSHARP` in this case.

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**